python-avd/pyavd/_eos_designs/structured_config/structured_config_generator.py

```diff
@@ -24,7 +24,21 @@ class StructCfgs:
 
     root: list[EosCliConfigGen] = field(default_factory=list)
     nested: EosCliConfigGen = field(default_factory=EosCliConfigGen)
-    list_merge_strategy: Literal["append", "replace"] = "append"
+    list_merge_strategy: Literal["append_unique", "append", "replace", "keep", "prepend", "prepend_unique"] = "append_unique"
```

Comment:
wondering if it is not worth make an enum type out of this at some point

Reply:
Absolutely. But not Today :)